src/main/java/wellnus/command/HelpCommand.java

```diff
@@ -95,8 +95,9 @@ public void printGeneralHelpMessage() {
     }
 
     /**
-     * Prints the help message for a given commandToSearch.
-     * If it does not exist,
+     * Prints the help message for a given commandToSearch. <br/>
+     * If the commandToSearch does not exist, helpCommand will print an unknown command
```

Comment:
Same here: I think renaming `helpCommand` to 'help' command would be clearer since there's no actual variable/attribute called `helpCommand`